packages/broker/src/StreamMetrics.ts

```diff
@@ -137,9 +143,11 @@ export class StreamMetrics {
                     this.report.network.bytesFromPeersPerSec = throttledAvg(this.report.network.bytesFromPeersPerSec, (metricsReport.metrics.WebRtcEndpoint.inSpeed as any).rate || 0)
                     this.report.network.connections = throttledAvg(this.report.network.connections, (metricsReport.metrics.WebRtcEndpoint.connections as any).rate || 0)
 
-                    if (metricsReport.metrics['broker/cassandra']) {
-                        this.report.storage.bytesWrittenPerSec = throttledAvg(this.report.storage.bytesWrittenPerSec, (metricsReport.metrics['broker/cassandra']) ? (metricsReport.metrics['broker/cassandra'].writeBytes as any).rate: 0)
-                        this.report.storage.bytesReadPerSec = throttledAvg(this.report.storage.bytesReadPerSec, (metricsReport.metrics['broker/cassandra']) ? (metricsReport.metrics['broker/cassandra'].readBytes as any).rate : 0)
+                    if (metricsReport.metrics['broker/cassandra']){
+                        const bytesWrittenPerSec = (metricsReport.metrics['broker/cassandra'].writeBytes as {rate: number}).rate ||  0
```

Comment:
As you mentioned "Assuming that data in readBytes and writeBytes is always of type {rate: number, ...}", this fallback `0` is most likely redundant